MaterialDesignThemes.Wpf/DialogHost.cs

```diff
@@ -432,6 +432,15 @@ public CornerRadius CornerRadius
             set => SetValue(DialogContentProperty, value);
         }
 
+        public static readonly DependencyProperty DialogContentUniformCornerRadiusProperty = DependencyProperty.Register(
+            nameof(DialogContentUniformCornerRadius), typeof(double), typeof(DialogHost), new PropertyMetadata(4d));
+
+        public double DialogContentUniformCornerRadius
+        {
+            get { return (double)GetValue(DialogContentUniformCornerRadiusProperty); }
+            set { SetValue(DialogContentUniformCornerRadiusProperty, value); }
```

Comment:
nit: I have been favoring using labda-bodied members for these.
```suggestion
            get => (double)GetValue(DialogContentUniformCornerRadiusProperty);
            set => SetValue(DialogContentUniformCornerRadiusProperty, value);
```

Reply:
And so have I, but I guess laziness got the best of me. This was my IDE generating the dependency property boiler-plate code for me 🤣